README.md

```diff
@@ -110,6 +110,12 @@ An array of eight categorical colors represented as RGB hexadecimal strings.
 
 An array of twelve categorical colors represented as RGB hexadecimal strings.
 
+<a href="#schemeTableau10" name="schemeTableau10">#</a> d3.<b>schemeTableau10</b> [<>](https://github.com/d3/d3-scale-chromatic/blob/master/src/categorical/Tableau10.js "Source")
+
+<img src="https://raw.githubusercontent.com/d3/d3-scale-chromatic/master/img/Tableau10.png" width="100%" height="40" alt="Tableau10">
+
+An array of ten categorical colors from [Tableau 10](https://www.tableau.com/about/blog/2016/7/colors-upgrade-tableau-10-56782) represented as RGB hexadecimal strings.
```

Comment:
This mention appears it would satisfy the Tableau legal team's requirement for Tableau citation.

Reply:
Maybe replacing the word `from` with `authored by Tableau as part of` would make it even more explicit.

Reply:
Just changed the sentence and pushed the change into the forked repo. Should I create new PR or is there a way to reopen this one?